Formula/python@3.9.rb

```diff
@@ -57,19 +57,20 @@ class PythonAT39 < Formula
   link_overwrite "Frameworks/Python.framework/Resources"
   link_overwrite "Frameworks/Python.framework/Versions/Current"
 
+  # Find bundled versions at https://github.com/python/cpython/blob/v#{version}/Lib/ensurepip/__init__.py
```

Comment:
is this comment accurate? asking because for 3.9.6, cpython shipped with setuptools 56.0.0 while we used 57.0.0

Reply:
I think we always just ship the latest versions here.

Reply:
Yes, latest. Lines 284-287 overwrite the versions in this file.

Reply:
thanks for the clarification, updated the comment